### PR TITLE
Initial release fixes

### PR DESCRIPTION
**Pull request by mrjackv**
This PR:
* Fixes the missing README pictures in the marketplace/vscode
* Adds language contribution point that allows the plugin to show up when searching by file extension
* Fix labels references not being found and not being ctrl-clickable

### CODE REVIEW

**Comment by aleclearmind** [comment]
Please split this commit into multiple commits. One for bumping version number and one per each fix.

**Comment by aleclearmind** [comment]
Please split this commit into multiple commits. One for bumping version number and one per each fix.

**Comment by aleclearmind** [comment]
This is a test comment

**Comment by mrjackv** [comment]
PR split

File: src/web/llvmir/lsp_model_provider.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ export class LspModelProvider {
                             }
                         } else if (am.groups["user"] !== undefined) {
                             const varName = am.groups["user"];
-                            if (lastFunction?.values.get(varName) !== undefined) {
+                            if (varName.startsWith("%") && lastFunction !== undefined) {
```
Comment (aleclearmind) [inline src/web/llvmir/lsp_model_provider.ts:127]: Do we handle globals whose initializer uses globals that are defined later?

Comment (mrjackv) [reply src/web/llvmir/lsp_model_provider.ts:127]: yes, a global will fail the `startsWith` check and so it will be added unconditionally to the users

**Comment by aleclearmind** [comment]
Merged! Thanks!

**Pull request merged**